platforms/software/dependency-management/src/main/java/org/gradle/internal/component/model/DefaultIvyArtifactName.java

```diff
@@ -65,7 +65,7 @@ public DefaultIvyArtifactName(String name, String type, @Nullable String extensi
     }
 
     @Override
-    public String toString() {
```

Comment:
🤔 Can we add back a toString that just calls getDisplayName?  I'm just thinking about what this looks like in the debugger.

Reply:
Nevermind

Reply:
Discussed. This was removed intentionally so people didn't use this method for behavior.